jsonrpc/server.go

```diff
@@ -426,6 +426,19 @@ func isNil(i any) bool {
 	return i == nil || reflect.ValueOf(i).IsNil()
 }
 
+func isNilOrEmpty(i any) (bool, error) {
+	if isNil(i) {
+		return true, nil
+	}
+
+	switch reflect.TypeOf(i).Kind() {
+	case reflect.Slice, reflect.Array, reflect.Map:
```

Comment:
Should we support string type as well 🤔 

Also - wdyt to expose it & `isNil()` publicly from `utils`?

Reply:
Yeah we could, since `IsNil()` is used on other places

Reply:
> Should we support string type as well 🤔

No, A direct string value is not valid in `params`.